tests/impl.cpp

```diff
@@ -1857,6 +1859,22 @@ bool test_mm_cvtps_epi32(const float _a[4])
     return validateInt32(ret, trun[0], trun[1], trun[2], trun[3]);
 }
 
+bool test_mm_testz_si128(const int32_t *_a, const int32_t *_b)
+{
+    __m128i a = _mm_load_si128((const __m128i *) _a);
+    __m128i b = _mm_load_si128((const __m128i *) _b);
+    int testz = 0;
+    int i;
+    for (i = 0; i < 2; i++) {
+        if ((((SIMDVec *) &a)->m128_u64[i] & ((SIMDVec *) &b)->m128_u64[i]) ==
```

Comment:
Replace with macro `vreinterpretq_nth_u64_m128i`.

Reply:
Macro `vreinterpretq_nth_u64_m128i` is defined in header `sse2neon.h`. If the test suite is built for x86 target, header `sse2neon.h` will not be included. Then, this macro does not effect accordinly.